utils.js

```diff
@@ -1,20 +1,39 @@
-import { $, cd } from 'zx'
 import path from 'path'
 import fs from 'fs'
 import { fileURLToPath } from 'url';
+import { execaCommand } from 'execa';
 
 export let root
 export let vitePath
 export let workspace
+export let cwd
+export let env
+
+function cd(dir){
+  cwd = path.resolve(cwd,dir);
+}
+
+async function $(literals, ...values) {
+  const cmd = literals.reduce((result,current,i)=> result+current+(values?.[i] != null ? `${values[i]}` : ''),'')
```

Comment:
feels like there should be an easier way to get the full string out of a tagged literal with dynamic values but here we are

Reply:
I didn't expect you will keep the same format, this is a great hack!